externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/operation/SparkOperationSuite.scala

```diff
@@ -33,10 +33,10 @@ import org.apache.spark.sql.types._
 import org.apache.kyuubi.Utils
 import org.apache.kyuubi.engine.spark.WithSparkSQLEngine
 import org.apache.kyuubi.engine.spark.shim.SparkCatalogShim
-import org.apache.kyuubi.operation.JDBCTests
+import org.apache.kyuubi.operation.HiveJDBCTests
 import org.apache.kyuubi.operation.meta.ResultSetSchemaConstant._
 
-class SparkOperationSuite extends WithSparkSQLEngine with JDBCTests {
+class SparkOperationSuite extends WithSparkSQLEngine with HiveJDBCTests {
```

Comment:
Do you set catalog implementation to hive in this test suite?

Reply:
For HiveJDBCTests, it is only test for hive catalog?

Reply:
hive is the default catalogImplementation
```
    val defaultCat = if (KyuubiSparkUtil.hiveClassesArePresent) "hive" else "in-memory"
    sparkConf.setIfMissing("spark.sql.catalogImplementation", defaultCat)
```

Reply:
Actually, hive catalog has more functionalities than in-memory catalog, I'm not sure if we need in-memory catalog test in both engine side and kyuubi server side.